docs/static/releasenotes.asciidoc

```diff
@@ -68,6 +69,83 @@ This section summarizes the changes in the following releases:
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
 
+[[logstash-8-16-0]]
+=== Logstash 8.16.0 Release Notes
+
+==== Announcing the new {ls} health report API
+
+The new health report API (`GET /_health_report`) is available with {ls} `8.16.0` and later,
+with indicators that are capable of detecting degraded status of configured pipelines and
+providing actionable insights https://github.com/elastic/logstash/pull/16520[#16520], https://github.com/elastic/logstash/pull/16532[#16532].
+
+For a more detailed view please check <<logstash-health-report-api>>.
+
+[[notable-8.16.0]]
+==== Notable issues fixed
+
+* When a conditional evaluation encounter an error in the expression, the event that triggered the issue is sent to pipeline's DLQ, if enabled for the executing pipeline. https://github.com/elastic/logstash/pull/16423[#16423]
+
+* Fixed the `if` statement expression evaluation against runtime errors that could crash the pipeline, just logging the offending event and continuing with the next in the batch. https://github.com/elastic/logstash/pull/16322[#16322]
+
+[[core-8.16.0]]
+==== Other changes to Logstash core
+
+* Added an escape hatch to prevent the new health API's status from affecting the status field of other API responses; by setting the `logstash.forceApiStatus` system property to `green`, you can force the other APIs to continue to use hard-coded green value. https://github.com/elastic/logstash/pull/16535[#16535]
```

Comment:
```suggestion
* Added an escape hatch to prevent the new Health API's status from affecting the top-level `status` field of Logstash's existing API responses. By setting the `logstash.forceApiStatus` system property to `green` you can force the other APIs to return the previous hard-coded `green` value. https://github.com/elastic/logstash/pull/16535[#16535]
```